### PR TITLE
Prepare README

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

#95 

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by Simply007** [comment]
Code samples need testing. Could be part of #101.

**Comment by gormal** [comment]
Changes look great! I have just a few comments/questions, also I will check code examples more thoughtfully.

**Comment by Simply007** [comment]
Samples should be handled in #96 resp. #101.

So I think we can merge this one right?

**Comment by gormal** [comment]
> Samples should be handled in #96 resp. #101.
> 
> So I think we can merge this one right?

There are some pending comments outside of code samples, it would be great to see what you think about them before we merge this PR.

File: README.md [hunk 1]
```diff
@@ -44,56 +41,76 @@ ManagementOptions options = new ManagementOptions
 ManagementClient client = new ManagementClient(options);
 ```
 
-Once you create a `ManagementClient`, you can start managing content in your project by calling methods on the client instance. See [Importing content items](#importing-content-items) for details.
+Once you create a `ManagementClient`, you can start managing content in your project by calling methods on the client instance.
 
 ### Codename vs. ID vs. External ID
 
-Most methods of the SDK accept an *Identifier* object that specifies which content item, language variant, or asset you want to perform the given operation on. There are 3 types of identification you can use to create the identifier:
```
[the remark this answers is not in the document]
Comment (Simply007) [reply README.md:51]: Validate does not need it for example. But I will rewrite the wording.

File: README.md [hunk 2]
```diff
@@ -629,17 +293,17 @@ Prerequisites:
...
 Optional:
-* [Visual Studio 2017](https://www.visualstudio.com/vs/) for full experience
-* or [Visual Studio Code](https://code.visualstudio.com/)
+
+- [Visual Studio 2017](https://www.visualstudio.com/vs/) for full experience
+- or [Visual Studio Code](https://code.visualstudio.com/)
 
 #### Tests
+
 Tests can run against Live endpoint or mocked filesystem. `TestUtils.TestRunType` specifies target environemnt for tests. Commit always with TestRunType.MockFromFileSystem. For updating mocked data use `TestUtils.TestRunType.LiveEndPoint_SaveToFileSystem` and before commit update `Data` directory with the content from `\Kentico.Kontent.Management.Tests\bin\Debug\net5.0\Data\`. When using `TestRunType.MockFromFileSystem`, at the build time, data from `Data` directory are copied to bin and tests are running against mocked data.
```
[the remark this answers is not in the document]
Comment (Simply007) [reply README.md:302]: We might be in the future. Currently I would expect a GitHub issue requesting the API key. I can cut that out to separate md file if you want.

File: README.md [hunk 3]
```diff
@@ -629,17 +293,17 @@ Prerequisites:
...
 
-##### For internal maintainers
-Note source project's got the `Backup` environment with the copy of the project. It might be used to restore if the source environment is corrupted. When changing project data, do not forget to update. or even better create a new environment with commit/issue identifier timestamp. 
+##### For contributors
 
-##### For external contributors
-Exported backup of the project created by [Template manager](https://kentico.github.io/kontent-template-manager/) is stored in `import-package.zip`. Note, items, language variants, sitemaps, assets,  might be imported with different `Id`s so it might be needed to update tests.
+Exported backup of the project created by [Template manager](https://kentico.github.io/kontent-template-manager/) is stored in `import-package.zip`. Note, items, language variants, sitemaps, assets, might be imported with different `Id`s so it might be needed to update tests.
```
[the remark this answers is not in the document]
Comment (Simply007) [reply README.md]: OK

**Comment by gormal** [comment]
Nice work. :)

**Review by gormal — APPROVED** [review]

**Pull request merged**